dipak/org/learn/training/assignment/Calculator.java

```diff
@@ -0,0 +1,10 @@
+package org.learn.training.assignment;
+
+import java.util.Scanner;
+
+public class Calculator {
+	
+	asdasdads
```

Comment:
what the hell is this?

Reply:
There is a problem in line 7 of ur code

Reply:
this is sometyhing./. none of your concern